{
  "timestamp": "2024-03-02T09:53:26.767+00:00",
  "status": 500,
  "error": "Internal Server Error",
  "trace": "java.util.concurrent.CompletionException: io.axoniq.axonserver.connector.AxonServerException: [AXONIQ-4003] Received exception while dispatching command\r\n\tat java.base/java.util.concurrent.CompletableFuture.encodeThrowable(CompletableFuture.java:332)\r\n\tat java.base/java.util.concurrent.CompletableFuture.completeThrowable(CompletableFuture.java:347)\r\n\tat java.base/java.util.concurrent.CompletableFuture$UniApply.tryFire(CompletableFuture.java:636)\r\n\tat java.base/java.util.concurrent.CompletableFuture.postComplete(CompletableFuture.java:510)\r\n\tat java.base/java.util.concurrent.CompletableFuture.completeExceptionally(CompletableFuture.java:2194)\r\n\tat io.axoniq.axonserver.connector.command.impl.CommandChannelImpl$CommandResponseHandler.onError(CommandChannelImpl.java:394)\r\n\tat io.grpc.stub.ClientCalls$StreamObserverToCallListenerAdapter.onClose(ClientCalls.java:481)\r\n\tat io.grpc.internal.ClientCallImpl.closeObserver(ClientCallImpl.java:574)\r\n\tat io.grpc.internal.ClientCallImpl.access$300(ClientCallImpl.java:72)\r\n\tat io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInternal(ClientCallImpl.java:742)\r\n\tat io.grpc.internal.ClientCallImpl$ClientStreamListenerImpl$1StreamClosed.runInContext(ClientCallImpl.java:723)\r\n\tat io.grpc.internal.ContextRunnable.run(ContextRunnable.java:37)\r\n\tat io.grpc.internal.SerializingExecutor.run(SerializingExecutor.java:133)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)\r\n\tat java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)\r\n\tat java.base/java.lang.Thread.run(Thread.java:1583)\r\nCaused by: io.axoniq.axonserver.connector.AxonServerException: [AXONIQ-4003] Received exception while dispatching command\r\n\t... 11 more\r\nCaused by: io.grpc.StatusRuntimeException: CANCELLED: RST_STREAM closed stream. HTTP/2 error code: CANCEL\r\n\tat io.grpc.Status.asRuntimeException(Status.java:537)\r\n\t... 10 more\r\n",
  "message": "io.axoniq.axonserver.connector.AxonServerException: [AXONIQ-4003] Received exception while dispatching command",
  "path": "/product"
}